neurons/remote_config.py

```diff
@@ -141,6 +142,7 @@ def load_and_get_config_values(self):
         self.blockchain_importance_weight = self.get_config_value('blockchain_importance_weight', 1)
         
         self.discovery_timeout = self.get_config_value('discovery_timeout', 100)
+        self.challenge_timeout = self.get_config_value('challenge_timeout', 3000)
```

Comment:
should be 10-20 seconds I think, otherwise we might get grifters

Reply:
timeout is in seconds or milliseconds?

Reply:
seconds

Reply:
plz let me know why `discovery_timeout` is set that high?

Reply:
legacy